### PR TITLE
Module 5

### PR DESCRIPTION
**Pull request by ElBaronDudnik**

### CODE REVIEW

File: error.log [hunk 1]
```diff
@@ -0,0 +1,1812 @@
+{
```
Comment (vladyslav-kurytskyi) [inline error.log:1]: It's better not commit log files

File: src/index.ts [hunk 2]
```diff
@@ -2,14 +2,27 @@ import express from 'express';
...
 const port = 3700;
+const { combine, timestamp, label, prettyPrint } = winston.format;
 
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
 userRoutes(app);
 groupRoutes(app);
 
+export const logger = winston.createLogger({
```
Comment (vladyslav-kurytskyi) [inline src/index.ts:17]: Consider moving logger to the separate file

**Review by vladyslav-kurytskyi — COMMENTED** [review]
Please add middleware which will log invoked methods (task 5.1)
Please add "uncaughtException" and "unhandledRejection" listeners to the process, you can add them right in the index.js file (task 5.2)

**Comment by ElBaronDudnik** [comment]
@vladyslav-kurytskyi could you review the changes again, I've updated it?

**Pull request merged**